cmake/CMakeLists.txt

```diff
@@ -82,6 +82,19 @@ set(protobuf_BUILD_TESTS OFF CACHE BOOL "Build protobuf tests" FORCE)
 set(NSYNC_ENABLE_TESTS OFF CACHE BOOL "Build protobuf tests" FORCE)
 set(ONNX_ML 1)
 
+if(onnxruntime_USE_OPENMP AND UNIX)
```

Comment:
why only for UNIX? Even on Windows, mklml.dll has a hard dependency on libiomp5md.dll.

Reply:
The dynamic link logic for UNIX and Windows are different. Windows specifies the dll to find the symbol, while Unix doesn't. 

---
In reply to: [288897615](https://github.com/microsoft/onnxruntime/pull/1131#discussion_r288897615) [](ancestors = 288897615)

Reply:
the issue is there on Windows as well....shouldn't be UNIX only.